website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,486 @@
+---
+title: Customize dbt models database, schema and alias
+id: customize-schema-alias
+description: "Learn how to properly adjust your generate_schema_name() and generate_alias_name() macros."
+displayText: Learn how to adjust your generate schema name and generate alias name.
+hoverSnippet: Learn how to adjust your generate schema name and generate alias name.
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+level: 'Advanced'
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+This guide explains how to customize the [schema](/docs/build/custom-schemas), [database](/docs/build/custom-databases) and [alias](/docs/build/custom-aliases) naming conventions in dbt to fit your data warehouse governance and design needs.
+When we develop dbt models and run or build them, objects get created in the data warehouse.
+
+In the rest of the article, for consistency, when we refer to database, we'll be referring to a Google Cloud Platform  project or to a Databricks catalog.
```

Comment:
Can we add the same verbiage and callout as on this page? https://docs.getdbt.com/docs/build/custom-databases

Reply:
Hiya @T-Dunlap 

Thanks for your comments and suggestions. i just wanted to ask, with this comment, would you like for me to redo the guide so that it's similar to: https://docs.getdbt.com/docs/build/custom-databases?

Kind Regards
Natalie

Reply:
Different warehouses have different names for "logical databases". The information in this document covers "databases" on Snowflake, Redshift, and Postgres, as well as "projects" on BigQuery or "catalogs" on Databricks Unity Catalog.

Reply:
I added a PR to include the Unity Catalog section here, but not sure how to get it approved: https://github.com/dbt-labs/docs.getdbt.com/pull/5788

Reply:
Hiya @T-Dunlap 

Thanks for clarifying that for me.
I'll add the callout for you.
Also, it looks like your PR https://github.com/dbt-labs/docs.getdbt.com/pull/5788 has been merged 

Kind Regards

Reply:
Since we're doing the callout, feel free to delete the section that says "In the rest of the article, for consistency, when we refer to database, we'll be referring to a Google Cloud Platform  project or to a Databricks catalog."